src/webextension/settings/actions.js

```diff
@@ -31,13 +33,15 @@ function localResetStarting(actionId) {
 }
 
 function localResetCompleted(actionId) {
+  telemetry.recordEvent("resetCompleted", "settings");
```

Comment:
The reset request and completion were captured as expected for me 👍 

![telemetry_data_and_update_metrics_md_and_add_events_by_irrationalagent_ _pull_request__342_ _mozilla-lockbox_lockbox-extension](https://user-images.githubusercontent.com/49511/32849653-053aa2b4-c9ed-11e7-9f8b-e2b603b737c7.png)